server/user.go

```diff
@@ -43,7 +43,8 @@ func (c *Connection) JiraAccountID() types.ID {
 }
 
 type ConnectionSettings struct {
-	Notifications bool `json:"notifications"`
+	Notifications bool  `json:"notifications"`
+	Watching      *bool `json:"watching"`
```

Comment:
We'll need to update display logic of settings to include this new setting. It needs to included added here:

https://github.com/mattermost/mattermost-plugin-jira/blob/6678caee71ef1a0c5c1ea86fc360ea27161702d3/server/user.go#L49-L55

Reply:
@mickmister  I've pushed a new commit for this change, could you please check it?

Reply:
In this PR, can we also add separate settings for assignee and mention notifications, as explained here https://github.com/mattermost/mattermost-plugin-jira/pull/705#discussion_r631444153? The `Reporter` notifications will not be included here however.